src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/DeliveryRequestResolver.java

```diff
@@ -44,8 +44,7 @@ public boolean canResolve(
         }
 
         Colony colony = (Colony) manager.getColony();
-        CitizenData freeDeliveryMan = colony.getCitizens()
-                                        .values()
+        CitizenData freeDeliveryMan = colony.getCitizenManager().getCitizens()
                                         .stream()
                                         .filter(c -> c.getCitizenEntity() != null && requestToCheck.getRequest().getTarget().isReachableFromLocation(c.getCitizenEntity().getLocation()))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 185 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)